joomla/joomla-cms
# [4.0] Fixed the parent item dropdown

Pull Request for Issue #21225  .

### Summary of Changes
Parent Item was always empty after changing the menu.
![image](https://user-images.githubusercontent.com/22980147/43527725-ec0e52ec-95af-11e8-9afe-49cb0a552548.png)

This PR fixes this problem.


### Testing Instructions
1. Click on system 
2. Click on global configuration
3. Click on system
4. Click on debug system to make it active
5. Save and close  
6. Create a new menu.
7. Open the menu item.
8.  Click on the menu.
![image](https://user-images.githubusercontent.com/22980147/43528053-c08bf1a0-95b0-11e8-89b1-642a78cbd704.png)
9. Check that the parent item is empty
![image](https://user-images.githubusercontent.com/22980147/43528135-eeb4d222-95b0-11e8-83f5-ef1b9bc3224a.png)
10. Apply patch
11. Change the menu item again
12. Check that the parent item is not empty
![image](https://user-images.githubusercontent.com/22980147/43528288-490b08d6-95b1-11e8-9cca-6bbb6a585f38.png)

### Expected result
Expected result is the parent item is not empty


### Actual result
Actual result is parent item is empty


### Documentation Changes Required
None
